prov/rxm/src/rxm_domain.c

```diff
@@ -893,6 +895,18 @@ int rxm_domain_open(struct fid_fabric *fabric, struct fi_info *info,
 					 FI_PEER, &peer_context);
 			if (ret)
 				goto err5;
+
+			attr.op = FI_MIN;
+			attr.datatype = FI_INT8;
+			attr.datatype_attr.count =1;
+			attr.datatype_attr.size =1;
+			attr.mode = 0;
+			for (int i = FI_BARRIER; i < FI_GATHER; i++) {
+				ret = fi_query_collective(rxm_domain->offload_coll_domain,
+					i, &attr, 0);
+				if (FI_SUCCESS == ret)
+					rxm_domain->offload_coll_mask |= BIT(i);
+			}
```

Comment:
Please move this into its own function.  Declare variables at the top of functions (makes them easier to find), and don't use reverse logic.  I.e. the check is whether 'ret' equals success, not success equals ret.

Reply:
Done. 
``static uint64_t get_capabilities_mask(struct fid_domain *domain)``